CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+## [4.1.0] - 2022-07-01
+### Added
+
+* Added PHP 8 to supported PHP versions in `composer.json`. Support for PHP >7.1 is maintained. No code changes or updates to non-dev dependencies are required.
+
+### Changed
+
+* Updated dev dependencies:
+    * Updated `phpspec/phpspec` to ^7.2 (^5.0 permitted for PHP 7.1 users)
+    * Updated `php-http/socket-client` to ^2.1.0
+
+See https://github.com/alphagov/notifications-php-client/pull/115 for further context
```

Comment:
Optional: We tend not to include changes for developer related things in our changelog as I think the main users are our external users (and we don't really have any of them contribute code or often).

Open to removing the changed section as I don't think many users will find it relevant.

Either way is fine, I'll leave you to make the call.